ubuntudesign/docs.ubuntu.com
# Include MAAS 1.9 docs in docs.ubuntu.com

docs.maas.io is going away. This is fine apart from that the 1.9 documentation which resides there
is for a currently supported version of MAAS, and so should be preserved.

We don't have the time to convert this to Markdown for inclusion in the new [MAAS docs](https://github.com/canonicalltd/maas-docs),
so as a quick-and-dirty hack we're simply downloading the documentation wholesale, with the following tweaks:

- Update links to static files so that it works in docs.ubuntu.com
- Change the link to "MAAS dev documentation" to "MAAS 1.9 documentation"
- Change the target of "Latest documentation" to /maas inside this very site
- Remove the outdated `os-support` page

As a separate task, the [navigation in MAAS docs](https://github.com/CanonicalLtd/maas-docs/blob/master/en/metadata.yaml#L1) should be updated to link to these docs.

Resolves #29 

QA
--

``` bash
./rebuild-docs
./run
```

Navigate to <http://127.0.0.1:8007/maas/1.9/en/> and check it looks fine, works fine, and that you can get back to the main MAAS docs.
